### ISSUE TITLE
onErrorImageURL is not working.

### ISSUE THREAD

**Issue by DannyNobul** [xiaolin/react-image-gallery#479]
I am trying to replace the images with the default image if the resource failed to load. However, the onErrorImageURL prop doesn't seem to be replacing the images even when I pass it any image url string .

```
import DefaultPropertyImage from "../../../../src_new/assets/images/default-property-image.png";

<ImageGallery
  items={this.getImages()}
  ref={this.imageCarouselRef}
 onErrorImageURL="https://www.google.ca/images/branding/googlelogo/2x/googlelogo_color_272x92dp.png"
  useBrowserFullscreen={false}
  showPlayButton={true} 
/>
```

**Comment by xiaolin** [comment]
How are testing this? Are you forcing an image error by providing an invalid link?

e.g.
```jsx
<ImageGallery
   items ={[{original: 'invalid_link'}]}
   onErrorImageURL="https://www.google.ca/images/branding/googlelogo/2x/googlelogo_color_272x92dp.png"
/>
```

**Comment by DannyNobul** [comment]
No I am putting the placeholder on the onImageErrorURL. For testing purposes I used the google image which is a valid link.

Daniel Kulangiev

Front End Developer

[signature_538216802]

Nobul Corporation
​​A 170 University Ave, 2nd Floor, Toronto, ON, M5H 3B3
O 416‑304‑9044 x 2020
M 647-464-0209
​_________________________________________________________
IMPORTANT: The contents of this email and any attachments are confidential.
​They are intended for the named recipient(s) only. If you have recieved this email
​by mistake, please notify the sender immediately and do not disclose the contents
​​to anyone or make copies thereof.

**Comment by xiaolin** [comment]
Right, but I'm wondering how you are forcing `onErrorImageURL` to be used? `onErrorImageURL` will only be set to the `src` of the slide if the `src` of the slide fails to load. Are you able to force a failure on your `original` slide image?

**Comment by DannyNobul** [comment]
Because some of the links comes back with a 404. For example:

https://nobul.com/cdn-cgi/image/width=1000,height=1000,format=auto/https://listingimages.nobul.com/CF_MFR533755712/1.jpeg<https://nobul.com/cdn-cgi/image/width=1000,height=1000,format=auto/https:/listingimages.nobul.com/CF_MFR533755712/1.jpeg>


Daniel Kulangiev

Front End Developer

[signature_1978522426]

Nobul Corporation
​​A 170 University Ave, 2nd Floor, Toronto, ON, M5H 3B3
O 416‑304‑9044 x 2020
M 647-464-0209
​_________________________________________________________
IMPORTANT: The contents of this email and any attachments are confidential.
​They are intended for the named recipient(s) only. If you have recieved this email
​by mistake, please notify the sender immediately and do not disclose the contents
​​to anyone or make copies thereof.

**Comment by xiaolin** [comment]
Hmm, then it should work I'll mark as bug and take a look into it.

**Comment by DannyNobul** [comment]
I appreciate it, thank you.

Daniel Kulangiev

Front End Developer

[signature_192732493]

Nobul Corporation
​​A 170 University Ave, 2nd Floor, Toronto, ON, M5H 3B3
O 416‑304‑9044 x 2020
M 647-464-0209
​_________________________________________________________
IMPORTANT: The contents of this email and any attachments are confidential.
​They are intended for the named recipient(s) only. If you have recieved this email
​by mistake, please notify the sender immediately and do not disclose the contents
​​to anyone or make copies thereof.

**Comment by xiaolin** [comment]
Fixed in version 1.0.6
